macros/macros.in

```diff
@@ -234,14 +234,14 @@ package or when debugging this package.\
 %_exec_prefix		%{_prefix}
 %_bindir		%{_exec_prefix}/bin
 %_sbindir		%{_exec_prefix}/sbin
-%_libexecdir		%{_exec_prefix}/libexec
+%_libexecdir		%{_exec_prefix}/lib
 %_datadir		%{_prefix}/share
 %_docdir		%{_datadir}/doc
 %_sysconfdir		/etc
 %_sharedstatedir	%{_prefix}/com
 %_localstatedir		/var
 %_lib			lib
-%_libdir		%{_exec_prefix}/%{_lib}
+%_libdir		%{_exec_prefix}/%{_lib}/%({__dpkg_architecture} -qDEB_HOST_MULTIARCH)
```

Comment:
@ascherer The other option I wanted to go with (to enable this selectively as an override) requires #52 to fixed first.

It would probably make sense for it to be done selectively, as multilib and multiarch behavior is different, and retains backward compatibility.